components/landing/RelevantArticle.tsx

```diff
@@ -1,12 +1,12 @@
 import { Box, Link, Text } from "@chakra-ui/react";
 import * as React from "react";
-import { Article } from "../../utils/loadArticles";
+import { RelevantArticle } from "../../api-functions/CMS/Contentful";
 
 type Props = {
-  article: Article;
+  article: RelevantArticle;
 };
 
-export function RelevantArticle({ article }: Props) {
+export function Article({ article }: Props) {
```

Comment:
Any particular reason for the rename? Imo, we should stick with RelevantArticle since it is a more accurate representation of how it is shown on the landing page.